css/popup.css

```diff
@@ -1,37 +1,99 @@
+:root {
+  --roar-blue: #164176;
+  --roar-pink: #fa759e;
+
+  --window-height: 300px;
+  --window-width: 350px;
+}
+
+html,
+body {
+  height: var(--window-height);
+  width: var(--window-width);
+  background-color: var(--roar-blue);
+  padding: 0;
+  margin: 0;
+}
+
 #app-container {
-  color: #164176;
-  min-height: 500px;
-  min-width: 450px;
-  padding: 16px;
+  min-height: calc(var(--window-height) - 10px);
+  min-width: calc(var(--window-width) - 10px);
+  padding: 5px;
 }
 
 header {
   display: flex;
   align-items: center;
   justify-content: center;
-  padding-bottom: 16px;
 }
 
-header > .main-logo {
+/* header > .main-logo {
   height: 64px;
   width: 64px;
 }
 
 header > h1 {
   font-size: 24px;
   margin-left: 24px;
-}
+} */
 
 main {
-  height: 100%;
+  min-height: calc(var(--window-height) - 10px);
+  min-width: calc(var(--window-width) - 10px);
+  display: flex;
+}
+
+.profile-img {
+  flex: 0 0 auto;
+  align-self: flex-start;
+  margin-right: 5px;
+}
+
+.twitter-interface {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.DraftEditor-root {
+  color: white;
+  width: 250px;
```

Comment:
Let's increase the font-size and make the first line appear centered vertically with the avatar.